### PR TITLE
Fixing typos Dowload -> Download

### PR DESCRIPTION
**Pull request by ardalis**

### CODE REVIEW

File: test/Microsoft.AspNetCore.Mvc.FunctionalTests/FileResultTests.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ public FileResultTests(MvcTestFixture<FilesWebSite.Startup> fixture)
         public async Task FileFromDisk_CanBeEnabled_WithMiddleware()
         {
             // Arrange & Act
-            var response = await Client.GetAsync("http://localhost/DownloadFiles/DowloadFromDisk");
+            var response = await Client.GetAsync("http://localhost/DownloadFiles/DownloadFromDisk");
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Mvc.FunctionalTests/FileResultTests.cs]: Ignore these

**Comment by pranavkm** [comment]
The tests seem to be still failing. I can have a look before merging this in.

**Comment by Eilon** [comment]
@pranavkm all yours.

**Comment by pranavkm** [comment]
Thanks for the PR!

**Pull request closed without merging**